src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/missingswitchdefault/InputMissingSwitchDefault.java

```diff
@@ -49,5 +49,9 @@ public void nestedSwitch2() {
             }
             default: break;
         }
+
+        switch(i) { // violation
```

Comment:
Needed to add this for coverage, but will not compile after Java 17.

Reply:
we do not need sources that are not compilable. Whole idea of Input file compilations is to make sure that we never executes on weird files and AST is always predictable. We removed a lot of code from Checks after enforcement of compilation.

Please share problems with coverage.

Reply:
This syntax compiles in all Java versions before Java 17. We have this test case (empty switch) in google style test, but not in actual unit tests for this check.

Reply:
False alarm, it in compiled file now. I thought it is non compiled.